### ISSUE TITLE
`RemoteRunnable.invoke` method accepting configurable as `kwargs` instead of `config`

### ISSUE THREAD

**Issue by alexjedi** [langchain-ai/langchainjs#3749]
Hello langchain team,

First and foremost, thank you for the fantastic technology that has enabled our team to experiment with agents and models in a more convenient format.

We are utilizing langchain python, fastapi, and pydantic, listening to backend requests. For the client side, we're using this library, TypeScript, and React.

We've encountered an issue where calling the invoke or stream method for RemoteRunnable, the configurable, in which we pass the necessary request parameters, gets sent as kwargs instead of as config, triggering an error in the agent.

```js
const chain = new RemoteRunnable({
  url: `{endpoint_url}`,
});
const output = await chain.invoke(
  {
    input:
      'Create a new contact named John Smith and link him to a new company, Acme.',
  },
  {
    configurable: {
      destination ,
      integration_id,
      user_id,
    },
  },
);

```

However, when we use python to send a request from the client, the configurable is passed in config, and the agent sends a valid response.

```python
remote_runnable.invoke(
  {"input": "Create a new contact named John Smith and link him to a new company, Acme."},
  {
    config={
      "configurable": {
        "destination": "{destination}",
        "integration_id": "{integration_id}",
        "user_id": "{user_id}",
      }
    },
  }
);
```

In the implementation of this method, I see that config will always remain empty. Could you please help us understand this issue and suggest a solution?

**Comment by bracesproul** [comment]
Hey @alexjedi thank you for catching this! I just opened a PR to fix #3751 and it'll be included in our next release (either tonight or tmro)

**Comment by alexjedi** [comment]
@bracesproul Wow, that's fast. Thank you so much!
